.github/workflows/integration_app_harness.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           SCREENSHOT_DIR: /tmp/screenshots
           REDIS_URL: ${{ matrix.state_manager == 'redis' && 'redis://localhost:6379' || '' }}
         run: |
-          poetry run pytest integration
+          poetry run pytest integration --ignore=integration/test_node_version.py
```

Comment:
we need a better way to skip this test, otherwise it's just going to fail all the time when running locally